unified_range/utils.py

```diff
@@ -95,8 +99,12 @@ def create_from_semver(semver):
     :return: VersionRange
     """
     operators = {"lt": "<", "lte": "<=", "gt": ">", "gte": ">="}
+    mvnoperators = {"mvnlt": ")", "mvnlte": "]", "mvngt": "(", "mvngte": "["}
 
-    if not any(op in semver for op in operators.values()):
+    if any(op in semver for op in mvnoperators.values()):
+        raise ValueError(
+            "Version ranges seems to already be maven version range")
+    elif not any(op in semver for op in operators.values()):
```

Comment:
Maybe change that to be `if` instead of `elif` - logically its not related to the first if statement